salt/_states/kubeconfig.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from base64 import b64encode
+
+__virtualname__ = 'metalk8s_kubeconfig'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def managed(name, client_cert_info, config_context, **kwargs):
+    """Generate kubeconfig file with identities for control plane components"""
+    # TODO: manage idempotency
```

Comment:
This is required for this PR to land, as far as I'm concerned.

Reply:
Yes, It is required for this PR